dockerHashing/src/main/scala/cromwell/docker/DockerInfoActor.scala

```diff
@@ -109,6 +109,7 @@ final class DockerInfoActor(
   }
 
   private def backpressure(commandContext: DockerInfoContext) = IO {
+    logger.warn(s"Backpressuring DockerInfoActor enqueue for image ${commandContext.dockerImageID.fullName}")
```

Comment:
Not a holdover; the DNS theory is only a theory and I still want more info

Reply:
TOL: Is this worth exposing as a metric?